CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ## [Unreleased]
 
+## [v0.16.4] - 2023-06-23
```

Comment:
@markus-liang-lrn @bhavya-shukla-lrn @markkellyeire I think we can probably disable this Codacy rule ("Lists should be surrounded by blank lines"), since it goes against our own convention in this file?